source/_components/media_player.cambridgeaudio.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Cambridge Audio (StreamMagic)"
+description: "Instructions on how to integrate Cambridge Audio network audio players (using their StreamMagic platform) into Home Assistant."
+date: 2018-05-10 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: cambridgeaudio.png
+ha_category: Media Player
+ha_iot_class: "Local Polling"
+ha_release: 0.68
```

Comment:
Home Assistant `v0.68` has already been shipped. Please update this to the next major version, which is currently `v0.73`.